src/common/state/db_client_table.h

```diff
@@ -65,4 +65,20 @@ typedef struct {
   void *subscribe_context;
 } DBClientTableSubscribeData;
 
+/*
+ * ==== Plasma manager heartbeats ====
+ */
+
+/**
+ * Start sending heartbeats to the plasma_managers channel. Each
+ * heartbeat contains this database client's ID. Heartbeats can be subscribed
+ * to through the plasma_managers channel. Once called, this "retries" the
+ * heartbeat operation forever, every HEARTBEAT_TIMEOUT_MILLISECONDS
+ * milliseconds.
```

Comment:
I see, all of this is to avoid a malloc? It feels cleaner to me to have the timer in the manager instead of in redis code. Especially since that's what we're doing with the local scheduler and we'll need to do it if we want to include load information or other information in the heartbeats.